labs/SettingsControls/src/SettingsCard/SettingsCard.xaml

```diff
@@ -236,72 +256,89 @@
                                         </VisualState.Setters>
                                     </VisualState>
                                 </VisualStateGroup>
-
-                                <VisualStateGroup x:Name="IconStates">
-                                    <VisualState x:Name="IconVisible">
+                                <VisualStateGroup x:Name="ContentAlignmentStates">
+                                    <VisualState x:Name="Right" />
+                                    <VisualState x:Name="Left">
+                                        <VisualState.StateTriggers>
+                                            <labs:IsEqualStateTrigger Value="{Binding ContentAlignment, RelativeSource={RelativeSource TemplatedParent}}"
+                                                                      To="Left" />
+                                        </VisualState.StateTriggers>
                                         <VisualState.Setters>
-                                            <Setter Target="PART_IconPresenter.Visibility" Value="Visible" />
+                                            <Setter Target="PART_HeaderIconPresenter.Visibility" Value="Collapsed" />
+                                            <Setter Target="PART_DescriptionPresenter.Visibility" Value="Collapsed" />
+                                            <Setter Target="PART_HeaderPresenter.Visibility" Value="Collapsed" />
+                                            <Setter Target="PART_ContentPresenter.(Grid.Column)" Value="0" />
+                                            <Setter Target="PART_ActionIconPresenter.(Grid.Row)" Value="0" />
+                                            <Setter Target="PART_ContentPresenter.HorizontalAlignment" Value="Left" />
                                         </VisualState.Setters>
                                     </VisualState>
-                                    <VisualState x:Name="IconCollapsed" />
-
-                                </VisualStateGroup>
-
-                                <VisualStateGroup x:Name="DescriptionStates">
-                                    <VisualState x:Name="DescriptionVisible">
+                                    <VisualState x:Name="RightWrapped">
+                                        <VisualState.StateTriggers>
+                                            <labs:ControlSizeTrigger MaxWidth="{StaticResource SettingsCardWrapThreshold}"
```

Comment:
Can you see if `ThemeResource` here causes this to work? See the conversation here: https://github.com/microsoft/microsoft-ui-xaml/issues/4247#issuecomment-902157843

@chingucoding was there a dedicated bug for this somewhere in the WinUI repo? I didn't see an obvious one bubble up in a search.

Reply:
Ah, in my https://github.com/microsoft/microsoft-ui-xaml/issues/6945 bug on template binding, I called out bug https://github.com/microsoft/microsoft-ui-xaml/issues/5457 as the closest about the ThemeResource issue, but I don't think it's quite the same scenario, maybe we need a dedicated issue?

Reply:
@michael-hawker I don't think there is a dedicated issue for that and I also wasn't able to find anything in that area.